src/commonMain/kotlin/exchange.dydx.abacus/state/model/TradingStateMachine.kt

```diff
@@ -494,6 +497,10 @@ open class TradingStateMachine(
                 }
             }
 
+            "/v4/compliance/geoblock" -> {
+                print("compliance")
```

Comment:
i think you left a debug print in here

Reply:
ah yea, this is in a testing-specific function, but will remove to avoid confusion 👍